datasets/text/text.py

```diff
@@ -55,19 +55,17 @@ def _split_generators(self, dl_manager):
 
     def _generate_tables(self, files):
         for i, file in enumerate(files):
-            text_file_reader = pd.read_csv(
-                file,
-                dtype={"text": str},
-                names=["text"],
-                header=None,
-                iterator=True,
-                chunksize=self.config.chunksize,
-                encoding=self.config.encoding,
-                sep="\n",
-            )
-            for j, df in enumerate(text_file_reader):
-                pa_table = pa.Table.from_pandas(df)
-                # Uncomment for debugging (will print the Arrow table size and elements)
-                # logger.warning(f"pa_table: {pa_table} num rows: {pa_table.num_rows}")
-                # logger.warning('\n'.join(str(pa_table.slice(i, 1).to_pydict()) for i in range(pa_table.num_rows)))
-                yield (i, j), pa_table
+            j = 0
```

Comment:
Not trying to be an *ss, but perhaps you could use more meaningful vars here? file_idx, batch_idx, something like that. So that others can read through the code more easily. Looking great though!

Reply:
Sure !

Reply:
Yes Bram is right, in particular since this script is expected to be used and read a lot!

Reply:
> Not trying to be an *ss, but perhaps you could use more meaningful vars here? file_idx, batch_idx, something like that. So that others can read through the code more easily. Looking great though!

Reply:
I changed to file_idx and batch_idx